apartment_application_service/urls.py

```diff
@@ -14,6 +15,10 @@
 
 urlpatterns = [
     path("admin/", admin.site.urls),
+    path(
+        "v1/",
+        include((apartment_urls, "apartment"), namespace="v1/apartments"),
```

Comment:
Same as the comment about /projects

Reply:
sales part has defined to `apartment/urls.py` so it's not needed here.